### ISSUE TITLE
typo causing issues.. trying to install requirements.txt but looking for reqs.txt

### ISSUE THREAD

**Issue by gateway** [melMass/comfy_mtb#119]
### Describe the bug

https://github.com/melMass/comfy_mtb/blob/6b993b84071bbb80ba1b8bd63576f31e35d05590/install.py#L370

```
# Install dependencies from requirements.txt
def install_dependencies(dry=False):
    parsed_requirements = get_requirements(here / "reqs.txt")
    if not parsed_requirements:
        return
    print_formatted(
        "Installing dependencies from reqs.txt...", "italic", color="yellow"
    )

    for requirement in parsed_requirements:
        import_or_install(requirement, dry=dry)
```

should be

```
# Install dependencies from requirements.txt
def install_dependencies(dry=False):
    parsed_requirements = get_requirements(here / "requirements.txt")
    if not parsed_requirements:
        return
    print_formatted(
        "Installing dependencies from reqs.txt...", "italic", color="yellow"
    )

    for requirement in parsed_requirements:
        import_or_install(requirement, dry=dry)
```

reqs.txt does not exist :) 

### Operating System

Linux

### Comfy Mode

In a custom virtual env (venv, virtualenv, conda...)

### Console output

```sh
Traceback (most recent call last):
  File "/notebooks/ComfyUI/custom_nodes/comfy_mtb/install.py", line 384, in <module>
    import requirements
ModuleNotFoundError: No module named 'requirements'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/notebooks/ComfyUI/nodes.py", line 1735, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/notebooks/ComfyUI/custom_nodes/comfy_mtb/__init__.py", line 31, in <module>
    from .endpoint import endlog
  File "/notebooks/ComfyUI/custom_nodes/comfy_mtb/endpoint.py", line 6, in <module>
    from .utils import backup_file, here, import_install, reqs_map, run_command, styles_dir
  File "/notebooks/ComfyUI/custom_nodes/comfy_mtb/utils.py", line 23, in <module>
    from .install import pip_map
  File "/notebooks/ComfyUI/custom_nodes/comfy_mtb/install.py", line 386, in <module>
    print_formatted("Installing requirements-parser...", "italic", color="yellow")
  File "/notebooks/ComfyUI/custom_nodes/comfy_mtb/install.py", line 127, in print_formatted
    encoded_header = header.encode(sys.stdout.encoding, errors="replace").decode(
AttributeError: 'Logger' object has no attribute 'encoding'

Cannot import /notebooks/ComfyUI/custom_nodes/comfy_mtb module for custom nodes: 'Logger' object has no attribute 'encoding'
Traceback (most recent call last):
  File "/notebooks/ComfyUI/nodes.py", line 1735, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/notebooks/ComfyUI/custom_nodes/rgthree-comfy/__init__.py", line 98, in <module>
    for file in glob.glob('*.py', root_dir=DIR_PY) + glob.glob('*.js', root_dir=os.path.join(DIR_DEV_WEB, 'js')):
TypeError: glob() got an unexpected keyword argument 'root_dir'
```

**Comment by melMass** [comment]
Hi,
You linked to a very old commit, it's doesn't work like this anymore there is no install step now

**Comment by melMass** [comment]
Ah no my bad it's not an old commit. But all this code isn't actually used anymore, requirements should auto install in the try block but somehow it doesn't. 

TLDR before I was using tensorflow for the FILM nodes and this introduced all sorts of issues regarding dependencies and the portable mode (you cannot build source wheels there), Comfy Manager auto install each extensions requirements.txt so to avoid this back then I was using reqs.txt and the install script to detect the env and use the approriate requirements.
This was all removed in favor of using requirements.txt and removing the problematic dependencies.
I'm in the process of writing proper doc for all this on the wiki as for instance some nodes (the optional ones on the readme) aren't working out of the box.

I'm going to remove this part from install.py but I think the other error will persist (Logger.encoding) what version of python are you on?

**Comment by melMass** [comment]
Removed in 4e07450bcabb0105b5610e52f7d4692ea07f9c1d, let me know
